src/tester/test-runner.ts

```diff
@@ -55,7 +55,7 @@ export default async function main(options: Options, nProcesses?: number, regexp
 
 	console.log("Installing dependencies...");
 
-	await nAtATime(nProcesses, allDependencies(allPackages, typings), async pkg => {
+	await nAtATime(nProcesses, typings, async pkg => {

```

Comment:
We got the dependencies as part of `getAffectedPackages`. So don't need to get them again.